packages/plugin-html-choice/src/index.ts

```diff
@@ -1,13 +1,17 @@
 import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
 
+import { version } from "../package.json";
+
 const info = <const>{
   name: "html-choice",
+  version: version,
   parameters: {
     /** The HTML string to be displayed */
     html_array: {
       type: ParameterType.HTML_STRING,
       pretty_name: "Stimulus",
-      default: undefined,
+      default: [],
```

Comment:
Is this the correct default? An `undefined` default usually means that there is no allowed default so the plugin throws an error if the parameter isn't specified.